examples/package.json

```diff
@@ -16,10 +16,10 @@
   "author": "Funkia",
   "license": "MIT",
   "dependencies": {
-    "@funkia/rudolph": "0.0.5",
+    "@funkia/rudolph": "0.0.6",
     "todomvc-app-css": "^2.1.2"
   },
   "devDependencies": {
-    "parcel-bundler": "^1.9.7"
+    "parcel-bundler": "^1.10.3"
```

Comment:
I think it's nicer to update dependencies in seperate commits when the updates are unrelated to the changes in the commit.